code/game/gamemodes/wizard/rightandwrong.dm

```diff
@@ -308,6 +308,9 @@
 		S.summons_received = randomizeswords
 	playsound(src,'sound/items/zippo_open.ogg', 50, 1)
 
+/mob/living/carbon/human
```

Comment:
Cease

Reply:
is this even legal

Reply:
It is. It's a bit dumb, but that's a valid way to solve the problem and not a very costly one either.
It's just very ad-hoc.

I can't think of a better idea right now.
Maybe have a list of ckeys that were already given a book of absorb, and check that ?
Even better, check if they know that spell or not.

Reply:
Checking the spell was what I originally planned to do, but wizards usually throw all their summons at once and it takes people a second or two to pick up the book and use it. If the wizard summons again before they use the book and get the spell then another useless book will spawn.

Reply:
do this with the mind instead